### ISSUE TITLE
"Bluetooth" takes up too much MID real estate

### ISSUE THREAD

**Issue by RobertDuncan42** [tedsalmon/BlueBus#192]
I don't see why the static "Bluetooth" word needs to display always. I would like to see an option for "BT", or my favorite, only display "Bluetooth" for the first n seconds (where n is some small integer) of a song, then just scroll it off the display and just show song info  until the next song starts.

**Comment by lennylen** [comment]
Agreed, I decided to modify 1.3.4 to include metadata on the screen without the Bluetooth message displaying. 

When nothing is playing it goes back to Bluetooth. 

I've also reversed the role of the play and pause button from current state to desired state. There is a bug where the word Paused stays on the screen when you press Pause button on the MID. Not sure how to fix it. It does go away when the song changes, so not a huge deal.

It's here if you'd like to try it:

https://github.com/lennylen/BlueBusMod/commit/6fb361f6d5a9a713d510203c3d4398ed045bb7ac

**Comment by tedsalmon** [comment]
> I don't see why the static "Bluetooth" word needs to display always. I would like to see an option for "BT", or my favorite, only display "Bluetooth" for the first n seconds (where n is some small integer) of a song, then just scroll it off the display and just show song info until the next song starts.

Originally, the BlueBus used the standard radio UI where the screen real estate is split into two, with enough space to print the word "Bluetooth" on the left and not much more. When I switched to the TEL UI, this carried over. Honestly, I have been considering using the radio UI again as there's too much emulation happening on the MID UI to keep things looking "normal". While doesn't mean this can't be changed, it does go against the factory convention of showing you what mode you're in. "BT" might make more sense, OR I can just bring it inline with the single line UI where "Bluetooth" is displayed only when nothing else is happening with the screen (i.e. no metadata, and no "Paused").

**Comment by lennylen** [comment]
> > I don't see why the static "Bluetooth" word needs to display always. I would like to see an option for "BT", or my favorite, only display "Bluetooth" for the first n seconds (where n is some small integer) of a song, then just scroll it off the display and just show song info until the next song starts.
> 
> 
> 
> OR I can just bring it inline with the single line UI where "Bluetooth" is displayed only when nothing else is happening with the screen (i.e. no metadata, and no "Paused").

Not sure about everyone else, but this is definitely my preferred solution for the MID.

**Comment by tedsalmon** [comment]
> Not sure about everyone else, but this is definitely my preferred solution for the MID.
My qualm with it is that it goes against convention. On the single line UI I have no option as the display is 11 characters wide.

Perhaps changing it to "BT" but pinning it to the left of the screen at all times would look less janky than having "Bluetooth" scroll.

**Comment by lennylen** [comment]
> > Not sure about everyone else, but this is definitely my preferred solution for the MID.
> 
> 
> 
> My qualm with it is that it goes against convention. On the single line UI I have no option as the display is 11 characters wide.
> 
> 
> 
> Perhaps changing it to "BT" but pinning it to the left of the screen at all times would look less janky than having "Bluetooth" scroll.
> 
> 

I get it. For me personally, it's not that important since I'm using BlueBus exclusively.. it's the only mode I need 😇. Again, that's me not everyone else. However, I'd assume a lot of people could be in the same boat.

Is there anywhere else it could be placed? I've made a suggestion before about placing BT beside the MODE button. Yes, it still goes against convention.. but personally, i'd much prefer to save that room for the metadata, even if it is only 2 characters.
